mappings/net/minecraft/screen/slot/SlotActionType.mapping

```diff
@@ -1 +1,23 @@
 CLASS net/minecraft/class_1713 net/minecraft/screen/slot/SlotActionType
+	FIELD field_7789 QUICK_CRAFT Lnet/minecraft/class_1713;
+		COMMENT Drags items between multiple slots. This is usually triggered by the player clicking and dragging between slots.
+		COMMENT
+		COMMENT <p>This action happens in 3 stages. Stage 0 signals that the drag has begun, and stage 2 signals that the drag has ended. In between multiple stage 1s signal which slots were dragged on.</p>
+		COMMENT
+		COMMENT <p>The stage is packed into the click data along with the mouse button that was clicked. See {@link net.minecraft.screen.ScreenHandler#packQuickCraftData(int, int) ScreenHandler.packQuickCraftData(int, int)} for details.</p>
+	FIELD field_7790 PICKUP Lnet/minecraft/class_1713;
+		COMMENT Performs a normal slot click. This can pickup or place items in the slot, possibly merging the cursor stack into the slot, or swapping the slot stack with the cursor stack if they can't be merged.
```

Comment:
```suggestion
		COMMENT Performs a normal slot click. This can pickup or place items in the slot, possibly merging the cursor stack into the slot,
		COMMENT or swapping the slot stack with the cursor stack if they can't be merged.
```